### PR TITLE
Asad/chore/site theme

### PR DESCRIPTION
**Pull request by am5815**
Added some theme updates and mobile responsiveness.

### CODE REVIEW

File: src/pages/Index.tsx [hunk 1]
```diff
@@ -37,9 +37,15 @@ const Index: React.FC = () => {
...
-          </h5>
+          <div className="max-w-lg">
+            <h5 className="text-center font-bold font-sans mb-12 max-w-7xl px-8 text-4xl md:px-12 md:text-5xl z-[0]">
+              Discover Mental Health Resources for Teens
+            </h5>
+            <p className="text-center font-sans mb-12 max-w-7xl px-8 text-md md:px-12 md:text-lg z-[0]">
+              A multifaceted approach to address the growing mental health
+              crisis among youth of color and LGBTQ youth
+            </p>
+          </div>
```
Comment (wenhwang97) [inline src/pages/Index.tsx:48]: I think maybe this section should be wider. The size right now is the same as those cards below.

File: src/components/IonicCard.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default function IonicCard({
         color={"tertiary"}
         style={{
           height: 375,
-          width: 400,
+          maxWidth: 400,
           borderRadius: 20,
         }}
       >
```
Comment (wenhwang97) [inline src/components/IonicCard.tsx:53]: Now they have the same height. Maybe add some space between the paragraph and the "Visit" button as well.

**Review by wenhwang97 — COMMENTED** [review]
Very colorful. Just some advices on sizing and spacing.

File: src/pages/Index.tsx (hunk 1 above)
Comment (am5815) [reply src/pages/Index.tsx:48]: The text is not too much in length and if its wider, its just one line which seems bland. I think maybe we can play with sizing and colors but for now, I think keeping the width small to it appears on multiple lines looks better.

File: src/components/IonicCard.tsx (hunk 2 above)
Comment (am5815) [reply src/components/IonicCard.tsx:53]: addressed in new commit

**Review by wenhwang97 — CHANGES_REQUESTED** [review]
I think the hardcoded height messed up a lot of components in the card. I just found a way to fix it, which should solve the issue on home page and resource page.

**Review by wenhwang97 — APPROVED** [review]

**Pull request merged**